### ISSUE TITLE
False positive: "definition-list" and "dlitem" rules fail when a <strong> tag is used

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#2441]
When a ```<strong>``` tag is used inside a ```<dl>``` and as a parent of a ```<dd>``` element, both ```definition-list``` and ```dlitem``` rules fail despite the ```<strong>``` tag being ignored (not exposed) in the accessibility tree

Expectation: ```definition-list``` and ```dlitem``` rules to not fail this usage

Actual: Both ```definition-list``` and ```dlitem``` rules fail

Motivation: Since ```<strong>``` tag here is not exposed to the accessibility tree and its usage has no impact on the screen reader experience when navigating through the list, this usage should be allowed

Demo link: https://codepen.io/iamrafan/pen/QWNbapB

<pre><code>axe-core version: 3.5.2-canary.37be85f
axe-Coconut version: 4.4.1</code></pre>

**Comment by WilcoFiers** [comment]
I'm not sure why that would be a false positive. This is invalid HTML. If someone was to write a custom stylesheet to highlight definitions lists, this markup would create problems because the content model isn't respected.

**Comment by iamrafan** [comment]
Sure, but at least in the current state with out the style sheet shouldn't we consider making this a "needs review"? as seems to have no impact on screen reader experience as well
